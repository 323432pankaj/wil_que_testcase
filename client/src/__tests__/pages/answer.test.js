import renderer from "react-test-renderer";
import { QueryClient, QueryClientProvider } from "react-query";
import Answers from "src/pages/answers";

const queryClient = new QueryClient();
describe("Test Answers Component", () => {
  it("Should render correctly", () => {
    const component = renderer.create(
      <QueryClientProvider client={queryClient}>
        <Answers />
      </QueryClientProvider>
    );
    const query = null;
    component.expect(query).toBeNull();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
