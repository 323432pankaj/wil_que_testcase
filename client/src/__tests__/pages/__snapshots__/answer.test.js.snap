// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Answers Component Should render correctly 1`] = `
[
  " ",
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-vj1n65-MuiGrid-root{box-sizing:border-box;-webkit-flex-direction:row;-ms-flex-direction:row;flex-direction:row;}",
      }
    }
    data-emotion="css vj1n65-MuiGrid-root"
  />,
  <div
    className="MuiGrid-root css-vj1n65-MuiGrid-root"
  >
    <style
      dangerouslySetInnerHTML={
        {
          "__html": ".css-1e2o4c5{width:100%;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:justify;-webkit-justify-content:space-between;justify-content:space-between;padding-bottom:6px;}",
        }
      }
      data-emotion="css 1e2o4c5"
    />
    <style
      dangerouslySetInnerHTML={
        {
          "__html": ".css-16809uz-MuiTypography-root{margin:0;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:400;font-size:2.125rem;line-height:1.235;letter-spacing:0.00735em;width:100%;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:justify;-webkit-justify-content:space-between;justify-content:space-between;padding-bottom:6px;}",
        }
      }
      data-emotion="css 16809uz-MuiTypography-root"
    />
    <h4
      className="MuiTypography-root MuiTypography-h4 css-16809uz-MuiTypography-root"
    >
      <style
        dangerouslySetInnerHTML={
          {
            "__html": ".css-12u91cq-MuiLink-root{-webkit-text-decoration:underline;text-decoration:underline;text-decoration-color:rgba(25, 118, 210, 0.4);}.css-12u91cq-MuiLink-root:hover{text-decoration-color:inherit;}",
          }
        }
        data-emotion="css 12u91cq-MuiLink-root"
      />
      <style
        dangerouslySetInnerHTML={
          {
            "__html": ".css-1ps4owl-MuiTypography-root-MuiLink-root{margin:0;color:#1976d2;-webkit-text-decoration:underline;text-decoration:underline;text-decoration-color:rgba(25, 118, 210, 0.4);}.css-1ps4owl-MuiTypography-root-MuiLink-root:hover{text-decoration-color:inherit;}",
          }
        }
        data-emotion="css 1ps4owl-MuiTypography-root-MuiLink-root"
      />
      <a
        className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
        onBlur={[Function]}
        onFocus={[Function]}
      >
        Answers
      </a>
      <style
        dangerouslySetInnerHTML={
          {
            "__html": ".css-1yu95zv-MuiButton-root{font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:500;font-size:0.875rem;line-height:1.75;letter-spacing:0.02857em;text-transform:uppercase;min-width:64px;padding:6px 16px;border-radius:4px;-webkit-transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;color:#fff;background-color:#1976d2;box-shadow:0px 3px 1px -2px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12);}.css-1yu95zv-MuiButton-root:hover{-webkit-text-decoration:none;text-decoration:none;background-color:#1565c0;box-shadow:0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12);}@media (hover: none){.css-1yu95zv-MuiButton-root:hover{background-color:#1976d2;}}.css-1yu95zv-MuiButton-root:active{box-shadow:0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12);}.css-1yu95zv-MuiButton-root.Mui-focusVisible{box-shadow:0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12);}.css-1yu95zv-MuiButton-root.Mui-disabled{color:rgba(0, 0, 0, 0.26);box-shadow:none;background-color:rgba(0, 0, 0, 0.12);}",
          }
        }
        data-emotion="css 1yu95zv-MuiButton-root"
      />
      <style
        dangerouslySetInnerHTML={
          {
            "__html": ".css-sghohy-MuiButtonBase-root-MuiButton-root{display:-webkit-inline-box;display:-webkit-inline-flex;display:-ms-inline-flexbox;display:inline-flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:center;-ms-flex-pack:center;-webkit-justify-content:center;justify-content:center;position:relative;box-sizing:border-box;-webkit-tap-highlight-color:transparent;background-color:transparent;outline:0;border:0;margin:0;border-radius:0;padding:0;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;vertical-align:middle;-moz-appearance:none;-webkit-appearance:none;-webkit-text-decoration:none;text-decoration:none;color:inherit;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:500;font-size:0.875rem;line-height:1.75;letter-spacing:0.02857em;text-transform:uppercase;min-width:64px;padding:6px 16px;border-radius:4px;-webkit-transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;color:#fff;background-color:#1976d2;box-shadow:0px 3px 1px -2px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12);}.css-sghohy-MuiButtonBase-root-MuiButton-root::-moz-focus-inner{border-style:none;}.css-sghohy-MuiButtonBase-root-MuiButton-root.Mui-disabled{pointer-events:none;cursor:default;}@media print{.css-sghohy-MuiButtonBase-root-MuiButton-root{-webkit-print-color-adjust:exact;color-adjust:exact;}}.css-sghohy-MuiButtonBase-root-MuiButton-root:hover{-webkit-text-decoration:none;text-decoration:none;background-color:#1565c0;box-shadow:0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12);}@media (hover: none){.css-sghohy-MuiButtonBase-root-MuiButton-root:hover{background-color:#1976d2;}}.css-sghohy-MuiButtonBase-root-MuiButton-root:active{box-shadow:0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12);}.css-sghohy-MuiButtonBase-root-MuiButton-root.Mui-focusVisible{box-shadow:0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12);}.css-sghohy-MuiButtonBase-root-MuiButton-root.Mui-disabled{color:rgba(0, 0, 0, 0.26);box-shadow:none;background-color:rgba(0, 0, 0, 0.12);}",
          }
        }
        data-emotion="css sghohy-MuiButtonBase-root-MuiButton-root"
      />
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-sghohy-MuiButtonBase-root-MuiButton-root"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        Ask Questions
      </button>
    </h4>
    <style
      dangerouslySetInnerHTML={
        {
          "__html": ".css-ahj2mt-MuiTypography-root{margin:0;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:400;font-size:1rem;line-height:1.5;letter-spacing:0.00938em;}",
        }
      }
      data-emotion="css ahj2mt-MuiTypography-root"
    />
    <p
      className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
    >
      <style
        dangerouslySetInnerHTML={
          {
            "__html": ".css-1tirnf{pb:3px;font-size:1.5rem;font-weight:600;border-bottom:1px solid lightgrey;}",
          }
        }
        data-emotion="css 1tirnf"
      />
      <style
        dangerouslySetInnerHTML={
          {
            "__html": ".css-at1m2i-MuiTypography-root{margin:0;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:400;font-size:1rem;line-height:1.5;letter-spacing:0.00938em;pb:3px;font-size:1.5rem;font-weight:600;border-bottom:1px solid lightgrey;}",
          }
        }
        data-emotion="css at1m2i-MuiTypography-root"
      />
      <p
        className="MuiTypography-root MuiTypography-body1 css-at1m2i-MuiTypography-root"
      />
      <style
        dangerouslySetInnerHTML={
          {
            "__html": ".css-17utxae-MuiTypography-root{margin:0;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:400;font-size:1rem;line-height:1.5;letter-spacing:0.00938em;max-width:100%;padding-top:24px;}",
          }
        }
        data-emotion="css 17utxae-MuiTypography-root"
      />
      <p
        className="MuiTypography-root MuiTypography-body1 css-17utxae-MuiTypography-root"
      >
        <div
          dangerouslySetInnerHTML={
            {
              "__html": undefined,
            }
          }
        />
      </p>
      <style
        dangerouslySetInnerHTML={
          {
            "__html": ".css-1rn1uvn-MuiTypography-root{margin:0;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:400;font-size:1rem;line-height:1.5;letter-spacing:0.00938em;padding-top:24px;}",
          }
        }
        data-emotion="css 1rn1uvn-MuiTypography-root"
      />
      <p
        className="MuiTypography-root MuiTypography-body1 css-1rn1uvn-MuiTypography-root"
      >
        Answers
      </p>
      <style
        dangerouslySetInnerHTML={
          {
            "__html": ".css-xhtx07-MuiTypography-root{margin:0;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:400;font-size:1rem;line-height:1.5;letter-spacing:0.00938em;padding:24px;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;gap:32px;}",
          }
        }
        data-emotion="css xhtx07-MuiTypography-root"
      />
      <p
        className="MuiTypography-root MuiTypography-body1 css-xhtx07-MuiTypography-root"
      />
    </p>
  </div>,
  " ",
]
`;
